README.md

```diff
@@ -173,34 +166,76 @@ using System;
 using System.Threading.Tasks;
 using SendGrid;
 
-namespace Example
+class Program
 {
-    internal class Example
+    static async Task Main()
     {
-        internal static async Task Main(string[] args)
-        {
-            var apiKey = Environment.GetEnvironmentVariable("NAME_OF_THE_ENVIRONMENT_VARIABLE_FOR_YOUR_SENDGRID_KEY");
-            var client = new SendGridClient(apiKey);
-            var queryParams = @"{
-                'limit': 100
-            }";
-            var response = await client.RequestAsync(method: SendGridClient.Method.GET,
-                                                     urlPath: "suppression/bounces",
-                                                     queryParams: queryParams);
-	    }
+        var apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
+        var client = new SendGridClient(apiKey);
+        var queryParams = @"{'limit': 100}";
+        var response = await client.RequestAsync(method: SendGridClient.Method.GET,urlPath: "suppression/bounces",
+        queryParams: queryParams).ConfigureAwait(false);
     }
 }
 ```
+
 ## Web Proxy
 
 ```csharp
-var apiKey = Environment.GetEnvironmentVariable("NAME_OF_THE_ENVIRONMENT_VARIABLE_FOR_YOUR_SENDGRID_KEY");
+var apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY");
```

Comment:
change to `SENDGRID_API_KEY`

Reply:
see https://github.com/sendgrid/sendgrid-csharp/pull/922#discussion_r430825298